src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -27,7 +28,9 @@ class DialogFullScreen extends Modal {
 
   static defaultProps = {
     open: false,
-    enableBackgroundUI: true
+    enableBackgroundUI: true,
+    onCancel: null,
+    showCloseIcon: true
   }
```

Comment:
I suggest to replace this static with:
```
DialogFullScreen.defaultProps = {...}
```
to be consistent

Reply:
fixed: 2a1d3f6